compiler/luci/import/src/ImporterEx.test.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/ImporterEx.h"
+
+#include <gtest/gtest.h>
+#include <sstream>
+
+TEST(ImporterEx, uses_default_error_handler_NEG)
```

Comment:
about changed to NEG, figuring out how this is negative...

Q) what is this test doing? can't catch by the code.
it would help adding some comments.

Reply:
This is a negative test in a way that it fails to import a model from an invalid path. At the same time it does use the default error handler.